### DISCUSSION TITLE
Are all the tteck proxmox ct lxc apps built and running without docker? Want to make certain that proxmox backup/restore works with tteck lxc apps.

### DISCUSSION THREAD

**Discussion by chrislawso** [tteck/Proxmox#2306] [Q&A] [answered]
<sub>Be cautious and thoroughly evaluate scripts and automation tasks obtained from external sources.</sub>
I recently found https://github.com/fdcastel/Proxmox-Automation identifying that when using proxmox backup the contents of `/var/lib/docker` will not be included in backups. I was going to test some docker in lxc apps and realized that the docker datadir will need be correctly specified.

I wanted to ask in advance before testing any of the tteck lxc apps has anyone had any issues performing proxmox backup and restore? Are most/all the tteck lxc apps built without docker? I looked at some of the tteck scripts and I did not notice most being based using docker for the apps.

Thanks

**Comment by tteck** [comment]
`/var/lib/docker` is included in my backups.

**Comment by tteck** [comment, marked as answer]
Here's the list of LXCs that use Docker:
```
Home Assistant Container LXC
Docker LXC
Dockge LXC
Umbrel LXC
CasaOS LXC
Runtipi LXC
```
